src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -224,7 +225,7 @@ protected static boolean areItemStacksEqual(ItemStack stackA, ItemStack stackB)
     }
 
     protected boolean setupAndConsumeRecipeInputs(Recipe recipe) {
-        int[] resultOverclock = calculateOverclock(recipe.getEUt(), getMaxVoltage(), recipe.getDuration());
+        int[] resultOverclock = calculateOverclock(recipe.getEUt(), recipe.getDuration());
```

Comment:
Returning an array to represent an arbitrary slice of data (where element 0 is one thing and element 1 is some other thing) feels really bad to me. Is this idiomatic?

Reply:
(It appears it is, nevermind)

Reply:
It is a list of string options. The first is a special value "OFF" and comes from the translation file, 
the others are idiomatic (they are the known enery tier names, I am not sure this is what you meant?) and are not translated.